src/printer.js

```diff
@@ -2505,16 +2505,21 @@ function printNode(path, options, print) {
         { shouldBreak }
       );
     }
-    case "entry":
-      return printAssignment(
-        node.key,
-        path.call(print, "key"),
-        " =>",
-        node.value,
-        path.call(print, "value"),
-        false,
-        options
-      );
+    case "entry": {
+      const ref = node.byRef ? "&" : "";
+      const unpack = node.unpack ? "..." : "";
+      return node.key
```

Comment:
Do I understand this correctly that this differentiation is needed because of PHP 7.4 array unpacking and no `key` is available when unpacking something?

Reply:
With the new parser version, `entry` objects are created for every array entry, not just for associative arrays anymore (see old behavior [here](https://astexplorer.net/#/gist/6999f9ae8437215dbc768e225f9346a1/0a16da17f1798e93541e66447d6af5bff4471a9b)). Also, `unpack` and `byRef` were added to the `entry` node for a generic way to describe the context of a specific entry.

So with the new version, we have to account for `entry`s that don't have a `key` (non-associative arrays).

Reply:
Alright, this explains my follow-up question as well (why we didn't previously differentiate beetween kinds of array entries). 😁 

What I discovered (but I need to create an issue over at the parser for that): It's possible to do `[...&$var]` which should cause a parser error instead.